File: Python/KodeProgram/IKepemimpinan.py
# IKepemimpinan.py
from abc import ABC, abstractmethod

class IKepemimpinan(ABC):
    """Interface untuk Hybrid/Multiple Inheritance."""
    @abstractmethod
    def ambil_keputusan_strategis(self):
        pass

    @abstractmethod
    def get_unit_kerja(self):
        pass

File: CPP/KodeProgram/IKepemimpinan.cpp
#include <string>

#ifndef IKEPEMIMPINAN_CPP
#define IKEPEMIMPINAN_CPP

class IKepemimpinan {
public:
    virtual ~IKepemimpinan() = default; 
    virtual std::string ambilKeputusanStrategis() const = 0;
    virtual std::string getUnitKerja() const = 0;
};

#endif // IKEPEMIMPINAN_CPP